Extensions/PlatformBehavior/platformruntimebehavior.ts

```diff
@@ -248,7 +248,7 @@ namespace gdjs {
       return this._yGrabOffset;
     }
 
-    static NORMALPLAFTORM = 0;
+    static NORMALPLATFORM = 0;
```

Comment:
I'm not sure I'm in favor of change this. There might be someone using this constant in a javascript event. Or maybe you can add something like this (not sure it works like that):
```js
static NORMALPLATFORM = 0;
/** @deprecated - Use  NORMALPLATFORM instead. */
static NORMALPLAFTORM = PlatformRuntimeBehavior.NORMALPLATFORM;
```